README.md

```diff
@@ -417,6 +418,11 @@ Thanks to all [contributors](https://github.com/zudochkin/awesome-newsletters/gr
 - [The Morning Paper](https://blog.acolyer.org/). An interesting/influential/important paper from the world of CS every weekday morning, as selected by Adrian Colyer.
 - [TechMeme](https://www.techmeme.com/). Techmeme is a aggregated, filtered, archiveable summary in near real-time of what is new and generating conversation in technology.
 
+## Leadership
+
+- [Software Lead Weekly](https://softwareleadweekly.com/). Insightful articles about company culture, leadership and building software in general.
```

Comment:
This is a good idea but this is already in `Miscellaneous` section of the list. Can we remove it from there?

Reply:
Done ! Sorry, I didn't see it was already there.